book/src/fundamentals/get-and-modify.md

```diff
@@ -10,9 +10,10 @@ world.run(|mut u32s: ViewMut<u32>| {
 });
 ```
 
-[`get`](https://docs.rs/shipyard/latest/shipyard/trait.Get.html#tymethod.get) will return an `&T` when used with a [`&View<T>`](https://docs.rs/shipyard/latest/shipyard/struct.View.html) and an `&mut T` with a [`&mut ViewMut<T>`](https://docs.rs/shipyard/latest/shipyard/struct.ViewMut.html). You can also get an `&T` from a [`ViewMut<T>`](https://docs.rs/shipyard/latest/shipyard/struct.ViewMut.html), which is why we have to explicitly mutably borrow `u32s`.
+[`get`](https://docs.rs/shipyard/latest/shipyard/trait.Get.html#tymethod.get) will return a `&T` when used with a [`&View<T>`](https://docs.rs/shipyard/latest/shipyard/struct.View.html) and a `&mut T` with a [`&mut ViewMut<T>`](https://docs.rs/shipyard/latest/shipyard/struct.ViewMut.html).
+You can also get a `&T` from a [`ViewMut<T>`](https://docs.rs/shipyard/latest/shipyard/struct.ViewMut.html), which is why we have to explicitly get a mutable borrow on `u32s`.
 
-For single views if you're sure the entity has the component you want, you can index into it:
+If you're certain that an entity has the desired component, you can access it directly via index when using single views:
```

Comment:
I think the single view should come first since it's very important.
```suggestion
When using a single view, if you're certain that an entity has the desired component, you can access it directly via index:
```